fdbcli/fdbcli.actor.cpp

```diff
@@ -472,11 +472,14 @@ void initHelp() {
 		"change cluster coordinators or description",
 		"If 'auto' is specified, coordinator addresses will be choosen automatically to support the configured redundancy level. (If the current set of coordinators are healthy and already support the redundancy level, nothing will be changed.)\n\nOtherwise, sets the coordinators to the list of IP:port pairs specified by <ADDRESS>+. An fdbserver process must be running on each of the specified addresses.\n\ne.g. coordinators 10.0.0.1:4000 10.0.0.2:4000 10.0.0.3:4000\n\nIf 'description=desc' is specified then the description field in the cluster\nfile is changed to desc, which must match [A-Za-z0-9_]+.");
 	helpMap["exclude"] =
-	    CommandHelp("exclude [no_wait] <ADDRESS>*", "exclude servers from the database",
+	    CommandHelp("exclude [FORCE] [permanent] [no_wait] <ADDRESS>*", "exclude servers from the database",
```

Comment:
I feel `permanent` is not intuitive in term of what it does.
Since this new functionality is to exclude a `failed` or `limp` server, I feel `[failed]` or `[limp]` is better than `[permanent]`.

No need to change it now. Let's think about if there is a better option name.